src/util/tr.js

```diff
@@ -815,22 +892,25 @@ const _tr = (msg, {
     }
     iframe.setAttribute('style', 'z-index: 2147483647; display:block; height: 100% !important; width: 100% !important;min-height:80px !important;border:0px !important; border-color:none !important;')
     msgDiv.appendChild(iframe)
-    const ifrm = (iframe.contentWindow) ? iframe.contentWindow : (iframe.contentDocument.document) ? iframe.contentDocument.document : iframe.contentDocument
-    const doc = ifrm.document
 
     // Dispatch event for interstitial/exit intent close
     const closeCampaign = new Event('CT_campaign_rendered')
     document.dispatchEvent(closeCampaign)
 
-    doc.open()
-    doc.write(html)
     if (targetingMsgJson.display['custom-editor']) {
-      appendScriptForCustomEvent(targetingMsgJson, doc)
+      html = appendScriptForCustomEvent(targetingMsgJson, html)
     }
-    doc.close()
+    iframe.srcdoc = html
 
-    const contentDiv = document.getElementById('wiz-iframe-intent').contentDocument.getElementById('contentDiv')
-    setupClickUrl(onClick, targetingMsgJson, contentDiv, 'intentPreview', legacy)
+    iframe.onload = () => {
+      window.addEventListener('message', event => {
+        if (event.data.action === 'getnotifData') {
```

Comment:
I do not want to run into runtime errors. Please add condition like `if (event?.data?.action === 'getNotifData')`